src/bootstrap/config/tests.rs

```diff
@@ -19,6 +19,8 @@ fn download_ci_llvm() {
     assert_eq!(parse_llvm(""), if_available);
     assert_eq!(parse_llvm("rust.channel = \"dev\""), if_available);
     assert!(!parse_llvm("rust.channel = \"stable\""));
+    assert!(parse_llvm("llvm.assertions = true \r\n build.build = \"x86_64-unknown-linux-gnu\" \r\n llvm.download-ci-llvm = \"if-available\""));
+    assert_eq!(parse_llvm("llvm.assertions = true \r\n build.build = \"aarch64-apple-darwin\" \r\n llvm.download-ci-llvm = \"if-available\""), false);
```

Comment:
```suggestion
    assert!(!parse_llvm("llvm.assertions = true \r\n build.build = \"aarch64-apple-darwin\" \r\n llvm.download-ci-llvm = \"if-available\""));
```
Asserting for `false` should be done in this way rather than asserting equality with `false`.

Reply:
Got it! I'll get back to these tests later today, since there are formatting issues and adding `build.build` didn't help.